cloud/aws/metrics/adot-config.yaml

```diff
@@ -0,0 +1,54 @@
+receivers:
+  prometheus:
+    config:
+      global:
+        scrape_interval: 15s
+        scrape_timeout: 10s
+      scrape_configs:
+        - job_name: 'civiform_scraper'
+          static_configs:
+            - targets: [0.0.0.0:9000]
+  awsecscontainermetrics:
+    collection_interval: 10s
+
+processors:
+  filter:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - ecs.task.memory.utilized
+          - ecs.task.memory.reserved
+          - ecs.task.cpu.utilized
+          - ecs.task.cpu.reserved
+          - ecs.task.network.rate.rx
+          - ecs.task.network.rate.tx
+          - ecs.task.storage.read_bytes
+          - ecs.task.storage.write_bytes
+
+exporters:
+  awsprometheusremotewrite:
+    endpoint: '${PROMETHEUS_WRITE_ENDPOINT}'
+    aws_auth:
+      region: us-east-1
```

Comment:
Hmmm. We need to parameterize this somehow to pass region from outside.

Reply:
done